schemaregistry/schemaregistry.md

```diff
@@ -0,0 +1,257 @@
+
+### 2.1. Schema Group
+
+A schema group is a named collection of schemas. Since a schema registry is
+often a resource with a scope greater than a single application and might even
+span multiple organizations, it is very useful to put a grouping construct
+around sets of schemas that are related either by ownership or by a shared
+subject matter context.
+
+Implementations of this specification MAY associate access control rules with
+schema groups. For instance, a user or a group of users might be given write
+access only to a particular schema group that their organization owns. If trade
+secret protection is required for an application or parts of it and the schema
+structure would give some of those away, read access to a group of schemas might
+likewise be restricted. 
+
+Access control rules at the schema group level MUST NOT limit visibility of the
+schema groups themselves to users authorized to enumerate schema groups at the
+registry level or to create new schema groups in order to create unambiguous
+transparency about which `id` have already been taken. An implementation MAY
+withhold all details other than the `id` attribute from a user not authorized
+to access the schema group.
+
+This specification does not define management constructs for such access control
+rules.
+
+### 2.1.1. Schema group attributes
+
+The data model for a schema group consists of three attributes:
+
+#### id
+
+- Type: `String`
+- Description: Identifies the schema group.
+- Constraints:
+  - REQUIRED
+  - MUST be a non-empty string
+  - MUST conform with RFC3986/3.3 `segment-nz-nc` syntax
+  - MUST be unique within the scope of the schema registry
+- Examples:
+  - mygroup
+  - my-group
+
+#### format
+
+- Type: `String`
+- Description: Defines the schema format managed by this schema group. The
+  formats supported by an implementation are specific to the implementation.
+  This specification does not mandate support for particular formats.
+- Constraints:
+  - OPTIONAL. If not set, each schema in the group MAY have multiple, concurrent
+    format representations. See XXXX
+  - MUST be a non-empty string, if present.
+- Examples:
+  - "avro"
+  - "json-schema"
+  - "xsd11"
+
+#### description
+
+- Type: `String`
+- Description: Explains the purpose of the schema group.
+- Constraints:
+  - OPTIONAL
+- Examples:
+  - "This group holds schemas for the fabulous example app."
+
+## 2.2. Schema and Schema Version
+
+Conceptually, a schema is a description of a data structure. Since data
+structures evolve over time, the schema describing them will also evolve over
+time. Therefore, a schema often has multiple versions.
+
+For simple scenarios, the API allows for version management to be automatic and
+transparent. Whenever a schema is updated, a new version number is assigned and
+prior schema versions are retained. The latest available schema is always the
+default version that is retrieved when the API is given just the schema `id`.
```

Comment:
s/when the API..../when the URL to the schema is given without the version specified/

Not a biggie, but I didn't want people to think that the schema ID could be used w/o the group ID too.